requirements.txt

```diff
@@ -4,108 +4,14 @@
 #
 #    pip-compile requirements.in
 #
-aiohttp==3.8.3
-    # via py-cord
 aiopg[sa]==1.4.0
-    # via -r requirements.in
-aiosignal==1.2.0
-    # via aiohttp
-async-timeout==4.0.2
-    # via
-    #   aiohttp
-    #   aiopg
-attrs==22.1.0
-    # via aiohttp
-build==0.9.0
-    # via pip-tools
-cachetools==5.2.0
-    # via google-auth
-certifi==2022.9.24
-    # via requests
-charset-normalizer==2.0.4
-    # via
-    #   aiohttp
-    #   requests
-click==8.1.3
-    # via pip-tools
-colorama==0.4.6
-    # via
-    #   build
-    #   click
-frozenlist==1.3.1
-    # via
-    #   aiohttp
-    #   aiosignal
-google-auth==2.14.0
-    # via
-    #   google-auth-oauthlib
-    #   gspread
-google-auth-oauthlib==0.7.1
-    # via gspread
-greenlet==2.0.0.post0
-    # via sqlalchemy
-gspread==5.6.2
-    # via -r requirements.in
-idna==3.3
-    # via
-    #   requests
-    #   yarl
 marshmallow==3.18.0
-    # via -r requirements.in
-multidict==6.0.2
-    # via
-    #   aiohttp
-    #   yarl
-numpy==1.23.4
-    # via -r requirements.in
-oauthlib==3.2.2
-    # via requests-oauthlib
-packaging==21.3
-    # via
-    #   build
-    #   marshmallow
-pep517==0.12.0
-    # via build
-pip-tools==6.12.1
-    # via -r requirements.in
-psycopg2-binary==2.9.5
-    # via
-    #   aiopg
-    #   sqlalchemy
-py-cord==2.2.2
```

Comment:
Something definitely went wrong with this requirements compilation. Several packages from requirements.in are marked as deleted, and it looks like dependencies aren't being pulled in.

Honestly, the fastest way to resolve this is probably to delete the entire requirements.txt file, then run `pip-compile requirements.in` to get a completely fresh set of requirements. Versions may jump up a bit, but I'm not too worried about that.